src/partials/hero.html

```diff
@@ -0,0 +1,31 @@
+<!-- Hero section -->
+<section class="hero">
+  <div class="container">  <!-- tu position: relative; i elementy rozmieszczone za pomocą position: absolute; -->
+    <h1 class="hero__header">ICE CREAM MADE WITH PASSION</h1>
```

Comment:
W htmlu nie wpisujemy tekstu dużymi literami tylko zrobimy to w scss text-transform: uppercase